update doc: make clear the required permission for metrics regards the finalizers (#2380)
operator-framework/operator-sdk: CHANGELOG.md

- Added scorecard test short names to each scorecard test to allow users to run a specific scorecard test using the selector flag. ([#1916](https://github.com/operator-framework/operator-sdk/pull/1916)
- Improve Ansible logs in the Operator container for Ansible-based Operators. ([#2321](https://github.com/operator-framework/operator-sdk/pull/2321))
- Added support for override values with environment variable expansion in the `watches.yaml` file for Helm-based operators. ([#2325](https://github.com/operator-framework/operator-sdk/pull/2325))

> - Added `deployments/finalizers` permission in the default roles scaffold by the project. ([#2380](https://github.com/operator-framework/operator-sdk/pull/2380))

Reviewer: ```suggestion
- Added `deployments/finalizers` permission to the default roles scaffold for Go projects. ([#2380](https://github.com/operator-framework/operator-sdk/pull/2380))
```
